### PR TITLE
Build db container and use it in check run

### PR DESCRIPTION
**Pull request by 2martens**

### CODE REVIEW

**Comment by 2martens** [comment]
The workflow should work for pull requests as well. But currently there is a confirmed bug with locally places reusable workflows. GitHub is investigating: https://github.com/github/feedback/discussions/10679.

File: .github/workflows/check-docker-run.yml [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
-        image: ghcr.io/hbtgmbh/salat:main
+        image: ghcr.io/hbtgmbh/salat:${{ inputs.salat_tag }}
         ports:
           - 8080:8080
+          - 8000:8000
+        env:
+          DATABASE_USERNAME: salattest
+          DATABASE_PASSWORD: salattest
+          DATABASE_HOST: db
+          DATABASE_NAME: salattest
```
Comment (KlausRicharz) [inline .github/workflows/check-docker-run.yml:27]: Database name has been changed to salat

**Review by KlausRicharz — CHANGES_REQUESTED** [review]
Change database name

File: .github/workflows/check-docker-run.yml (hunk 1 above)
Comment (KlausRicharz) [reply .github/workflows/check-docker-run.yml]: habe ich kurz selbst gemacht

**Review by KlausRicharz — APPROVED** [review]

**Pull request merged**